spec/unit/generator/xcconfig/aggregate_xcconfig_spec.rb

```diff
@@ -241,20 +241,20 @@ def pod_target(spec, target_definition)
             @xcconfig.to_hash['OTHER_SWIFT_FLAGS'].should.include '$(inherited) "-D" "COCOAPODS"'
           end
 
-          it 'sets EMBEDDED_CONTENT_CONTAINS_SWIFT when there is swift' do
+          it 'sets ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES when there is swift' do
```

Comment:
For tests, I think there should be 2 sets: 1 set stubs the @swift_version to be 2.2, and the other stubs it to be 2.3. In one set of tests, it'll set `ALWAYS_EMBED_SWIFT_LIBRARIES`, and in the other it's set the older key. Since we're going to support both, we should probably have tests that check for both.